src/simulations/tests/set_generator_test.py

```diff
@@ -407,13 +412,30 @@ def test_sequentially_correlated_all_previous_generator_raise_error(self):
     rs = np.random.RandomState(1)
     with self.assertRaises(ValueError):
       _ = set_generator.SequentiallyCorrelatedSetGenerator(
-          order='not_implemented', correlated_sets='all', universe_size=30,
+          order='not_implemented', correlated_sets='all',
           shared_prop=0.2, set_sizes=[10] * 3, random_state=rs)
     with self.assertRaises(ValueError):
       _ = set_generator.SequentiallyCorrelatedSetGenerator(
-          order='random', correlated_sets='not_implemented', universe_size=30,
+          order='random', correlated_sets='not_implemented',
           shared_prop=0.2, set_sizes=[10] * 3, random_state=rs)
 
+  @parameterized.parameters(
+      (set_generator.CORRELATED_SETS_ALL,),
+      (set_generator.CORRELATED_SETS_ONE,))
+  def test_sequentially_correlated_generator_overlap_size_larger_than_set_size(
+      self, correlation_type):
+    sc_gen = set_generator.SequentiallyCorrelatedSetGenerator(
+        order=set_generator.ORDER_ORIGINAL, correlated_sets=correlation_type,
+        shared_prop=0.5, set_sizes=[10, 1],
+        random_state=np.random.RandomState())
```

Comment:
You sure you don't want a constant seed here?

Reply:
Added a constant seed.